docgen/generate-docs.js

```diff
@@ -116,6 +121,35 @@ function fixLinks(file) {
   });
 }
 
+/** 
+ * Adds links to external documentation for type aliases that
+ * reference an external library.
+ * 
+ * @param data File data to add external library links to.
+ */
+function addTypeAliasLinks(data) {
+  const htmlDom = new JSDOM(data);
+  const fileTags = htmlDom.window.document.querySelectorAll(".tsd-signature-type");
+  fileTags.forEach(tag => {
+    for (const type of typeMap.keys()) {
```

Comment:
Why not something simpler and more efficient like:

```
const mapping = typeMap[tag.textContent];
if (mapping) {
   // Add <a> tag
}
```

Reply:
I still think you should only replace full qualified external names like `firebase.auth.UserRecord`.